search/test_utils.py

```diff
@@ -18,8 +19,14 @@ def test_es_connection(self):
     def test_create_and_delete_indicator(self):
         indexer = ElasticsearchIndexer()
         indicator = Indicator.objects.create(name="TestIndicator")
-        indexer.delete_indicator(indicator.pk)
+        indexer.delete_indicator(indicator)
 
+    """
+    Disabled because an indicator object needs to exist in database to add a workflowlevel1 many to many relation
```

Comment:
But what happens for old indicators that were not added to Elasticsearch and they are not found? This is why I think that test made sense.

Reply:
Yes, the test made sense. And it still does, it is just not possible to set it up with the current flow. Because once the dummy object for the test is created it is also added to Elastic on save. If i don't save the object i won't be able to add a workflowlevel1 relation which is necessary for the delete to have the org association. Otherwise it won't find the right index.
I'm open for ideas.

Reply:
You can use the mock library to overwrite the elasticsearch save function and do nothing instead.

In python 3 comes by default. With the version we have you need to install it in requirements-ci and requirements-dev.
https://pypi.python.org/pypi/mock